libs/web/settings/data-access/src/lib/store/meta-reducers/init-settings.reducer.ts

```diff
@@ -1,15 +0,0 @@
-import { ActionReducer, INIT, UPDATE } from '@ngrx/store';
-import { LocalStorageService } from '../../services';
-import { SettingsState } from '../settings.reducer';
-
-export function initSettingsFromLocalStorage(
-  reducer: ActionReducer<SettingsState>
-): ActionReducer<SettingsState> {
-  return (state, action) => {
-    const newState = reducer(state, action);
-    if ([INIT.toString(), UPDATE.toString()].includes(action.type)) {
```

Comment:
IMO it is not neccessary to listen for the INIT action.
`StoreModule.forRoot` dispatches the INIT action.
The `SettingsModule` is registered after `StoreModule.forRoot(rootReducers)` was running already.
Therefore the INIT action was dispatched before this metaReducer is registered.

Also UPDATE is dispatched whenever a new reducer is added to the Store (e.g. through a lazy loaded module with StoreModule.feature). That means that the code inside the `if` will potentially be called many times. But I believe, reading the localStorage settings and assign it to the state should only happen once.